packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -238,6 +238,8 @@ async function normalizeTransactions(
     trans.account = acctId;
     trans.payee = await resolvePayee(trans, payee_name, payeesToCreate);
 
+    trans.category = trans.category || null;
```

Comment:
Nullish coalescing would be better here to avoid any potential issues with falsy values

```suggestion
    trans.category = trans.category ?? null;
```